networking/ipv6.md

```diff
@@ -152,66 +141,31 @@ then the following changes will ensure IPv6 operation.
-is easy to remedy that by launching an image, making appropriate changes
-to its configuration files, taking a snapshot, and then using that
-snapshot thereafter instead of the original image.
-
-For example, starting from an Ubuntu cloud image, the following
-changes will suffice to meet the requirements just listed.
-
--   In `/etc/network/interfaces.d/eth0.cfg`, add:
-
-        iface eth0 inet6 dhcp
-                accept_ra 1
-
--   In `/sbin/dhclient-script`, add at the start of the script:
-
-        new_ip6_prefixlen=128
-
--   In `/etc/sysctl.d`, create a file named `30-eth0-rs-delay.conf` with
-    contents:
-
-        net.ipv6.conf.eth0.router_solicitation_delay = 10
-
-For CentOS, these additions to a cloud-init script have been reported to be effective:
-
-	runcmd:
-	- sed -i -e '$a'"IPV6INIT=yes" /etc/sysconfig/network-scripts/ifcfg-eth0
-	- sed -i -e '$a'"DHCPV6C=yes" /etc/sysconfig/network-scripts/ifcfg-eth0
-	- sed -i '/PATH/i\new_ip6_prefixlen=128' /sbin/dhclient-script
-	- systemctl restart network
+#### Enable dual stack, OpenShift
 
+TBD - operator steps  
 
-### Enabling IPv6 support in {{site.prodname}}
+#### Enable IPv6-only, OpenShift
 
-In OpenStack, IPv6 connectivity requires defining an IPv6 subnet, in
-each Neutron network, with:
+TBD - operator steps  
 
--   the IPv6 address range that you want your VMs to use
--   DHCP enabled
--   IPv6 address mode set to DHCPv6 stateful.
+#### Configure the Kubernetes control plane to operate over IPv6
 
-We suggest initially configuring both IPv4 and IPv6 subnets in each
-network. This allows handling VM images that support only IPv4 alongside
-those that support both IPv4 and IPv6, and allows a VM to be accessed
-over IPv4 in case this is needed to troubleshoot any issues with its
-IPv6 configuration.
+(Optional). Depending on your networking set up, you may want to configure the Kubernetes control plane to operate over IPv6 for dual stack or IPv6 only.
+ 
+To configure Kubernetes components, enable IPv6 using the following flags.  
 
-In principle, though, we are not aware of any problems with configuring
-and using IPv6-only networks in OpenStack.
+| Component                   | **Flag**                                      | **Value/Content**                                            |
+| --------------------------- | --------------------------------------------- | ------------------------------------------------------------ |
+| **kube-apiserver**          | `--bind-address` or `--insecure-bind-address` | Set to the appropriate IPv6 address or `::` for all IPv6 addresses on the host. |
+|                             | `--advertise-address`                         | Set to the IPv6 address that nodes should use to access the `kube-apiserver`. |
+|                             | `--service-cluster-ip-range`                  | Set to an IPv6 CIDR that will be used for the Service IPs. The DNS service address must be in this range. |
```

Comment:
Delete row, because this is about workloads, not Kubernetes own control plane.

Reply:
delete.